imports/client/components/PuzzlePage.tsx

```diff
@@ -99,7 +103,7 @@ const DEBUG_SHOW_CALL_STATE = false;
 
 const tabId = Random.id();
 
-const FilteredChatFields: ('_id' | 'puzzle' | 'text' | 'sender' | 'timestamp')[] = ['_id', 'puzzle', 'text', 'sender', 'timestamp'];
+const FilteredChatFields: ('_id' | 'puzzle' | 'text' | 'content' | 'sender' | 'timestamp')[] = ['_id', 'puzzle', 'text', 'content', 'sender', 'timestamp'];
 type FilteredChatMessageType = Pick<ChatMessageType, typeof FilteredChatFields[0]>
```

Comment:
I know you didn't write it like this, but I think this would be better phrased as

```suggestion
const FilteredChatFields = ['_id', 'puzzle', 'text', 'content', 'sender', 'timestamp'] as const;
type FilteredChatMessageType = Pick<ChatMessageType, typeof FilteredChatFields[number]>
```

Reply:
Fixed in c22660e87c59e080e44c30f43aaa30059a093c01.  `as const` is magical.